test/src/toga_test/app.py

```diff
@@ -0,0 +1,12 @@
+import toga
+
+
+class TogaTest(toga.App):
+    def startup(self):
+        self.main_window = toga.MainWindow(title=self.formal_name)
+        self.main_window.content = toga.Box()
```

Comment:
> It might be worth adding a placeholder text label here letting the user know that this is a test suite, and they should probably be using `--test`

Fixed by merging https://github.com/mhsmith/toga/pull/1.

> We could possibly use this as a HTML-style [ACID test](https://www.acidtests.org) - a layout that has a very specific visual appearance that lets us do an easy sanity check of visual properties that can't be easily tested.

Do you mean using the the placeholder text for this, or something more complex? Either way, I think the best plan is to pay attention to what bug reports we get for each widget after we've fully tested it, and use that as guidance for what this approach is covering well or not so well.

Reply:
I was thinking something more complex. I don't have anything particular in mind; I was just thinking that it was real estate that we could use for a purpose, and a "visual layout that is known to be problematic" seemed like a useful purpose. However, as you say - we can use bug reports to guide this.